components/layouts/main/view.tsx

```diff
@@ -0,0 +1,29 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+
+export default function View() {
+    const { asPath } = useRouter();
+    
+    return asPath.includes('printers') ? 
+        (<div className="border-y border-y-pnc-grey pt-1.5 mt-[13px]">
+            <nav aria-label="secondary-navigation">
+                <ul className="flex gap-9">
+                    {[
+                        { path: "list", label: "List view" },
+                        { path: "assign", label: "Assign view" },
+                    ].map(({ path, label }) => (
+                    <li>
+                        <Link href={path}>
+                        <a className="text-sm tracking-wide font-bold">
+                            {label}
+                        </a>
+                        </Link>
+                    </li>
+                    ))}
+                </ul>
+            </nav>
+            <div className={`h-0 w-0 border-l-[6px] border-r-[6px] border-b-[6px] border-pnc-white border-b-pnc-grey 

```

Comment:
This implementation to mark the currently active sub-nav-item does not scale well, and is not centered perfectly for `Assign view`.

The common way to solve this is to use an `:after` pseudo-element on the `<a>` tag.